newsroom/ui_config_async.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional, Dict, Any
+from superdesk.core.module import Module
+from superdesk.core.mongo import MongoResourceConfig
+from superdesk.core.resources import ResourceModel, ResourceModelConfig
+from superdesk.core.resources.service import AsyncResourceService
+from content_api import MONGO_PREFIX
+
+
+class UiConfig(ResourceModel):
+    _id: str
+    preview: Optional[Dict[str, Any]] = None
+    details: Optional[Dict[str, Any]] = None
+    list: Optional[Dict[str, Any]] = None
+    advanced_search_tabs: Optional[Dict[str, Any]] = None
+    multi_select_topics: Optional[bool] = None
+    search: Optional[bool] = None
+    enable_global_topics: Optional[bool] = None
+    open_coverage_content_in_same_page: Optional[bool] = None
+    subnav: Optional[Dict[str, Any]] = None
+    init_version: Optional[int] = None
+
+
+class UiConfigResourceService(AsyncResourceService[UiConfig]):
+    async def get_section_config(self, section_name: str) -> Dict[str, Any]:
+        """Get the section config"""
+        config = await self.find_by_id(section_name)
+
+        if not config:
+            return {}
+        return config.dict()
```

Comment:
If you include the args `by_alias=True, exclude_unset=True`, then aliased fields will be properly converted to their field name, and any fields that are not set will not be included.